website/docs/docs/build/udfs.md

```diff
@@ -0,0 +1,191 @@
+---
+title: "User-defined functions"
+description: "Learn how to add user-defined functions (UDFs) to your dbt projects."
+id: "udfs"
+---
+
+# User-defined functions <Lifecycle status="beta" />
+
+:::info
+UDFs are not yet supported in the dbt Fusion engine.
+:::
+
+User-defined functions (UDFs) enable users to define and register custom functions in your warehouse. Like [macros](/docs/build/jinja-macros), UDFs promote code reuse. They run natively in the warehouse so you can reuse the same logic in tools outside dbt.
```

Comment:
Changing user to you and wonder if we can add some more of the business value here. cc @QMalcolm to check my technical accuracy.
```suggestion
User-defined functions (UDFs) enable you to define and register custom functions in your warehouse. Like [macros](/docs/build/jinja-macros), UDFs promote code reuse, but they run natively in the warehouse so you can reuse the same logic in tools outside dbt, such as BI tools, data science notebooks, and more. 

UDFs are particularly valuable for sharing logic across multiple tools, standardizing complex business calculations, improving performance for compute-intensive operations (since they're compiled and optimized by your warehouse's query engine), and version controlling custom logic within your dbt project.
```

Reply:
Looks good to me - my one suggestion would be to say "They are objects in the warehouse" instead of "They run natively in the warehouse"